mindsdb/integrations/handlers/sendinblue_handler/sendinblue_tables.py

```diff
@@ -58,4 +60,47 @@ def get_email_campaigns(self, **kwargs):
         connection = self.handler.connect()
         email_campaigns_api_instance = sib_api_v3_sdk.EmailCampaignsApi(connection)
         email_campaigns = email_campaigns_api_instance.get_email_campaigns(**kwargs)
-        return [email_campaign for  email_campaign in email_campaigns.campaigns]
\ No newline at end of file
+        return [email_campaign for  email_campaign in email_campaigns.campaigns]
+              
+    def delete(self, query: ast.Delete) -> None:
+        """
+        Deletes an email campaign from Sendinblue.
+
+        Parameters
+        ----------
+        query : ast.Delete
+           Given SQL DELETE query
+
+        Returns
+        -------
+        None
+
+        Raises
+        ------
+        ApiException
+            If an error occurs when calling Sendinblue's API
+        """
+        delete_statement_parser = DELETEQueryParser(query)
+        where_conditions = delete_statement_parser.parse_query()
+
+        email_campaigns_df = pd.json_normalize(self.get_email_campaigns())
+
+        delete_query_executor = DELETEQueryExecutor(
+            email_campaigns_df,
+            where_conditions
+        )
+
+        email_campaigns_df = delete_query_executor.execute_query()
+        campaign_ids = email_campaigns_df['id'].tolist()
+        self.delete_email_campaigns(campaign_ids)
+
+    def delete_email_campaigns(self, campaign_ids: List[Text]) -> None:
+        connection = self.handler.connect()
+        email_campaigns_api_instance = sib_api_v3_sdk.EmailCampaignsApi(connection)
+
+        for campaign_id in campaign_ids:
+            try:
+                email_campaigns_api_instance.delete_email_campaign(campaign_id)
+                logger.info(f'Email Campaign {campaign_id} deleted')
+            except ApiException as e:
+                logger.error(f"Exception when calling EmailCampaignsApi->delete_email_campaign: {e}\n")           
```

Comment:
Instead of logger, can you raise an exception here?

Reply:
Also, please remove this error here instead raise an exception.

Reply:
@Sekhar-Kumar-Dash As @tcchawla has said, please raise the exception here. When running this query on MindsDB Cloud, if an exception is not raised, the user will not know something went wrong because they cannot see the log message.